packages/arb-token-bridge-ui/src/components/TransferPanel/TokenApprovalDialog.tsx

```diff
@@ -82,22 +76,6 @@ export function TokenApprovalDialog(props: TokenApprovalDialogProps) {
     return `${ethFeeText} ${usdFeeText}`.trim()
   }, [ethFeeText, usdFeeText])
 
-  const tokenLogo = useMemo(() => {
```

Comment:
Moving this to a shared component. This component is used multiple times in the design so we will reuse it.
<img width="163" alt="Screenshot 2024-02-13 at 12 11 08" src="https://github.com/OffchainLabs/arbitrum-token-bridge/assets/47932951/adedf565-9347-4a7f-a566-ace046e0a8fb">